packages/platforms/browser/lib/auto-instrumentation/page-load-phase-spans.ts

```diff
@@ -12,30 +12,34 @@ export const instrumentPageLoadPhaseSpans = (
   route: string,
   parentContext: SpanContext
 ) => {
-  function createPageLoadPhaseSpan (name: string, startTime: number, endTime: number) {
+  function createPageLoadPhaseSpan (phase: string, startTime: number, endTime: number) {
```

Comment:
wonder if it's worth making this a union? 

`type Phase = 'Unload' | 'Redirect' | 'LoadFromCache'` etc...